designs/senya/src/bottom.mjs

```diff
@@ -0,0 +1,149 @@
+import { base } from './base.mjs'
+import { waistlineHeight } from './shared.mjs'
+import { ringsectorPlugin } from '@freesewing/plugin-ringsector'
+
+function senyaBottom({
+  utils,
+  sa,
+  points,
+  Path,
+  paths,
+  Snippet,
+  snippets,
+  options,
+  measurements,
+  macro,
+  part,
+}) {
+  let topCircumverence = measurements.underbust
+  let fullLength =
+    measurements.hpsToWaistBack -
+    waistlineHeight(
+      measurements.waistToUnderbust,
+      measurements.hpsToWaistBack,
+      options.lowerWaistline
+    ) +
+    measurements.waistToHips
+  let angle = 180 * options.circleRatio
+  let radiusMidriffTop = topCircumverence / utils.deg2rad(angle) / 2
+
+  if (angle > 90 && sa) {
+    const maxAngle = utils.rad2deg(Math.atan(radiusMidriffTop / sa))
+    if (angle > 90 + maxAngle) angle = 90 + maxAngle
+  }
+
+  const radiusHem = radiusMidriffTop + fullLength * options.lengthBonus
+
+  const ids = macro('ringsector', {
+    angle: angle,
+    insideRadius: radiusMidriffTop,
+    outsideRadius: radiusHem,
+    rotate: true,
+  })
+  const pathId = ids.paths.path
+  paths.seam = paths[pathId].clone().addClass('fabric')
+  paths[pathId].hide()
+
+  for (const [shortId, uid] of Object.entries(ids.points)) {
+    points[shortId] = points[uid].copy()
+  }
+
+  points.gridAnchor = points.in2Flipped.clone()
+
+  macro('cutonfold', {
+    from: points.in2Flipped,
+    to: points.ex2Flipped,
+    grainline: true,
+  })
+  points.logo = points.in2FlippedRotated.shiftFractionTowards(points.ex2FlippedRotated, -0.1)
+  snippets.logo = new Snippet('logo', points.logo)
+
+  points.title = points.in2FlippedRotated.shiftFractionTowards(points.ex2FlippedRotated, -0.5)
+  macro('title', { at: points.title, nr: 3, title: 'bottom' })
+
+  macro('sprinkle', {
+    snippet: 'notch',
+    on: ['in1Rotated', 'gridAnchor'],
```

Comment:
Is `in1Rotated` where you intended the notch to be? 
![Screenshot 2024-01-19 at 12 02 11 PM](https://github.com/freesewing/freesewing/assets/109869956/d0a2c561-6bc8-421f-81f7-d250b70ed9a4)